tests/ui/cbmc_checks/signed-overflow/expected

```diff
@@ -7,12 +7,5 @@ Failed Checks: attempt to calculate the remainder with a divisor of zero
 Failed Checks: attempt to calculate the remainder with overflow
 Failed Checks: attempt to shift left with overflow
 Failed Checks: attempt to shift right with overflow
-Failed Checks: arithmetic overflow on signed addition
-Failed Checks: arithmetic overflow on signed subtraction
-Failed Checks: arithmetic overflow on signed multiplication
-Failed Checks: shift distance is negative
-Failed Checks: shift distance too large
-Failed Checks: shift operand is negative
 Failed Checks: arithmetic overflow on signed shl
-Failed Checks: shift distance is negative
-Failed Checks: shift distance too large
```

Comment:
Isn't that absence of failing checks concerning?

Reply:
I think those checks were redundant, i.e. each failure condition was being checked twice.

Reply:
Sorry, I should've added that to the call-out. Yes, @zhassan-aws is correct that these checks were redundant.

These operations that we changed are checked operations, which means that the codegen adds an overflow check already. Since the implementation does not invoke the arithmetic operation directly, CBMC doesn't add its own overflow check anymore.

That said, I think I deleted some of the shift errors by accident. They shouldn't be affected.

Reply:
Ok, thanks, I should have looked at more context in that file (which has all those codegen-added overflow checks also as "Failed Checks".